### PR TITLE
[opt](invert index) use lowercase by default

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/olap/inverted_index_parser.h [hunk 1]
```diff
@@ -49,6 +49,9 @@ struct InvertedIndexCtx {
 
 using InvertedIndexCtxSPtr = std::shared_ptr<InvertedIndexCtx>;
 
+const std::string INVERTED_INDEX_PARSER_YES = "true";
```
Comment (xiaokang) [inline be/src/olap/inverted_index_parser.h:52]: TRUE/FALSE

File: be/src/olap/rowset/segment_v2/segment_iterator.cpp [hunk 2]
```diff
@@ -1234,7 +1234,7 @@ Status SegmentIterator::_init_inverted_index_iterators() {
         if (_inverted_index_iterators[cid] == nullptr) {
             RETURN_IF_ERROR(_segment->new_inverted_index_iterator(
                     _opts.tablet_schema->column(cid),
-                    _opts.tablet_schema->get_inverted_index(unique_id), _opts,
+                    _segment->_tablet_schema->get_inverted_index(unique_id), _opts,
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/segment_iterator.cpp:1237]: add comment to explain why use segment tablet_schema

File: be/src/olap/tablet_schema.cpp [hunk 3]
```diff
@@ -616,6 +616,12 @@ void TabletIndex::to_schema_pb(TabletIndexPB* index) const {
     for (const auto& kv : _properties) {
         (*index->mutable_properties())[kv.first] = kv.second;
     }
+
+    // lowercase by default
+    if (!_properties.contains(INVERTED_INDEX_PARSER_LOWERCASE_KEY)) {
```
Comment (xiaokang) [inline be/src/olap/tablet_schema.cpp:621]: It may be better to set default value true on fe.

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Pull request merged**